sycl/source/detail/scheduler/scheduler.hpp

```diff
@@ -450,12 +450,18 @@ class Scheduler {
   ~Scheduler();
 
 protected:
+  // TODO: after switching to C++17, change std::shared_timed_mutex to
+  // std::shared_mutex
+  using RWLockT = std::shared_timed_mutex;
+  using ReadLockT = std::shared_lock<RWLockT>;
+  using WriteLockT = std::unique_lock<RWLockT>;
+
   /// Provides exclusive access to std::shared_timed_mutex object with deadlock
   /// avoidance
   ///
-  /// \param Lock is an instance of std::unique_lock<std::shared_timed_mutex>
+  /// \param Lock is an instance of WriteLockT
   /// class
```

Comment:
Non-blocking comment: This comment is not informative, maybe it is not needed.
Lock is obviously an instance of WriteLockT because the next line has: WriteLockT &Lock

Reply:
Fixed.